### PR TITLE
BOAC-3181: Adds setting to enable drop-in advising by department

### PR DESCRIPTION
**Pull request by lyttam**
https://jira.ets.berkeley.edu/jira/browse/BOAC-3181

### CODE REVIEW

File: config/test.py [hunk 1]
```diff
@@ -30,6 +30,8 @@
 
 DATA_LOCH_RDS_URI = 'postgres://boac:boac@localhost:5432/boac_loch_test'
 
+DEPARTMENTS_SUPPORTING_DROP_INS = ['QCADV']
+
```
Comment (johncrossman) [inline config/test.py]: @lyttam - Do we need the `test` config above?  You've nicely used `override_config` instead. Plus, it's nice to have minimal reliance on this test config file.

**Comment by johncrossman** [comment]
PR looks good, although Travis test failures need attention. Thanks @lyttam

File: config/test.py (hunk 1 above)
Comment (lyttam) [reply config/test.py]: @johncrossman I'll remove it as part of the fix for the failing tests.

**Comment by lyttam** [comment]
Ready for another look, @johncrossman.

**Comment by johncrossman** [comment]
Thanks @lyttam

**Pull request merged**